ComputationalArt.py

```diff
@@ -0,0 +1,256 @@
+    """
+    H = math.degrees(cmath.phase(c))
+    V = 1
+    #+(math.atan(abs(c))/(math.pi/2))
+    return colorsys.hsv_to_rgb(H,1,V)
+
+
+def generate_art_real(filename, c1 =[255,0,0], c2 = [0,255,0],c3 =[0,0,255], mins = [2,2,2], maxes = [10,10,10], frequ_dict_real1 = default_real_weights,frequ_dict_real2 = default_real_weights,frequ_dict_real3 = default_real_weights, x_size=350, y_size=350):
+    """ Generate computational art and save as an image file.
+
+        filename: string filename for image (should be .png)
+        x_size, y_size: optional args to set image dimensions (default: 350)
+    """
+    # Functions for red, green, and blue channels - where the magic happens!
+    weighted_choices1 = build_choice(frequ_dict_real1)
+    weighted_choices2 = build_choice(frequ_dict_real2)
+    weighted_choices3 = build_choice(frequ_dict_real3)
+    color_function_one = build_random_function_real(mins[0],maxes[0],weighted_choices1)
+    color_function_two = build_random_function_real(mins[1],maxes[1],weighted_choices2)
+    color_function_three = build_random_function_real(mins[2],maxes[2],weighted_choices3)
+
+    # Create image and loop over all pixels
+    im = Image.new("RGB", (x_size, y_size))
+    pixels = im.load()
+    for i in range(x_size):
+        for j in range(y_size):
+            x = remap_interval(i, 0, x_size, -1, 1)
+            y = remap_interval(j, 0, y_size, -1, 1)
+            color_1 =[color_map(evaluate_random_function_real(color_function_one, x, y),c1[0]),color_map(evaluate_random_function_real(color_function_one, x, y),c1[1]),color_map(evaluate_random_function_real(color_function_one, x, y),c1[2])]
+            color_2 =[color_map(evaluate_random_function_real(color_function_two, x, y),c2[0]),color_map(evaluate_random_function_real(color_function_two, x, y),c2[1]),color_map(evaluate_random_function_real(color_function_two, x, y),c2[2])]
+            color_3 =[color_map(evaluate_random_function_real(color_function_three, x, y),c3[0]),color_map(evaluate_random_function_real(color_function_three, x, y),c3[1]),color_map(evaluate_random_function_real(color_function_three, x, y),c3[2])]
+            pixels[i, j] = (
+                    color_1[0]+color_2[0]+color_3[0],
+                    color_1[1]+color_2[1]+color_3[1],
+                    color_1[2]+color_2[2]+color_3[2]
+                    )
+
+    im.save(filename)
+
+
+def generate_art_imaginary(filename, maxes=[5],mins=[2],frequ_dict_imaginary = default_imaginary_weights, x_size=350, y_size=350):
+    """ Generate computational art and save as an image file.
+
+        filename: string filename for image (should be .png)
+        x_size, y_size: optional args to set image dimensions (default: 350)
+    """
+    weighted_choices = build_choice(frequ_dict_imaginary)
+    if maxes[0] >8:
+        maxes[0] = 8
+    if mins[0] > maxes[0]:
+            mins[0] = maxes[0]
+    HS = build_random_function_imaginary(mins[0],maxes[0],weighted_choices)
```

Comment:
Avoid naming your variable with all capital letters. A variable name that is all capitals is reserved for a constant.